File: scholar-web/src/main/webapp/app/model/administration/settings/course/CourseSearch.js
Ext.define('scholar.model.administration.settings.course.CourseSearch', {
	extend : 'Ext.data.Model',
	idProperty: 'id',
	fields : [ {
		name : 'id',
		type : 'int'
	},{
		name : 'courseCode',
		type : 'string'
	}, {
		name : 'courseName',
		type : 'string'
	}, {
		name : 'lastChange',
		type : 'date',
		dateFormat : 'n/j h:ia'
	} ],

});

File: scholar-business/src/main/java/com/varun/scholar/business/data/entities/administration/settings/CourseSearch.java
package com.varun.scholar.business.data.entities.administration.settings;

import java.util.Date;

import javax.persistence.Entity;
import javax.persistence.Temporal;
import javax.persistence.TemporalType;

import com.varun.scholar.business.data.entities.AbstractEntity;


@Entity
public class CourseSearch extends AbstractEntity {
	String courseCode;
	String courseName;
	@Temporal(TemporalType.DATE)
	Date lastChange;
}
